docs/release-notes/highlights-6.3.0.asciidoc

```diff
@@ -1,9 +1,140 @@
 [[release-highlights-6.3.0]]
-== 6.3.0 release highlights
+== 6.3.0 Release Highlights
 ++++
 <titleabbrev>6.3.0</titleabbrev>
 ++++
 
 coming[6.3.0]
 
-See also <<breaking-changes>> and <<release-notes-6.3.0>>.
\ No newline at end of file
+Each release of {kib} brings new features and product improvements.
+Here are the highlights of the features that were added in 6.3 and
+how the user experience improved.
+
+Refer to the <<release-notes-6.3.0,6.3.0 Release Notes>> for a list bug
+fixes and other changes.
+
+[float]
+=== License management and {xpack} code
+
+Starting with version 6.3, all of the free {xpack} features, such
+as monitoring, tile maps, Grok Debugger, and Search Profiler,
+ship with the default distribution of {kib}.  When you install {kib},
+it generates a Basic license with no expiration date.
+
+You can view the status of your license, install a new license,
+or start a 30-day trial of https://www.elastic.co/subscriptions[Platinum features]
+in *Management > License Management*.
+The trial enables you to try out features such as
+security, machine learning, alerting,
+graph capabilities,
+and more.
+
+At the end of the trial period, your license reverts to a Basic license.
+To continue using the Platinum features, you can purchase a
+https://www.elastic.co/subscriptions[subscription] and update
+you license.
+
+[role="screenshot"]
+image::images/management_basic_license.png[]
+
+[float]
+=== Autocomplete and simple syntax
+
+{kib}'s experimental autocomplete and simple syntax features
+can help you create queries. To turn on these features,
+click *Options* on the right of the query bar
+and opt in.
+
+Once enabled, just start typing in the query bar,
+and you'll get suggestions for fields,
+values for keyword fields, and query operators.
+For information on the simplified, easier-to-use syntax, see
+<<kuery-query, {kib} Query Language Enhancements>>.
+
+
+
+[float]
+=== Return to recently viewed content
+
+{kib} now keeps track of your recently viewed
```

Comment:
nit: It looks like new lines are created arbitrarily throughout this doc, and I'm not sure why that is. I think we usually hard wrap at 80 characters.